### ISSUE TITLE
Sliced Wasserstein distance  vs Wasserstein distance 

### ISSUE THREAD

**Issue by shiqingzhangCSU** [scikit-tda/persim#51]
I think your job is very meaningful and exciting！But I am a little confused about the description of Sliced Wasserstein distance.
**"The general idea is to compute an approximation of the Wasserstein distance by computing the distance in 1-dimension repeatedly and use the results as a measure. "**
Does this mean that Sliced Wasserstein distance is faster than Wasserstein in terms of time complexity?
If not, what is the difference between these two distances in this library?
Thank you for your attention！
Looking forward to your reply!

**Comment by bartolsthoorn** [comment]
Yes the Sliced Wasserstein distance is much faster to compute and it approximates the Wasserstein distance. You can increase the parameter `M` in `persim.sliced_wasserstein(PD1, PD2, M=50)` to make it more precise, at a computational cost.
